packages/x-components/src/x-modules/popular-searches/store/__tests__/getters.spec.ts

```diff
@@ -5,7 +5,7 @@ import Vue from 'vue';
 import Vuex, { Store } from 'vuex';
```

Comment:
I think that in this file the interface `PopularSearchesRequest` should be used instead of `SuggestionsRequest`.